krug-2023-03/plugin/markdown/example.html

```diff
@@ -0,0 +1,129 @@
+<!doctype html>
```

Comment:
Not sure if you guys are still relying on this `plugin` structure (I just copied the whole directory from other folder). Let me know if it's needed.